bundles/org.eclipse.osgi.services/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,7 @@ Bundle-Vendor: %eclipse.org
 Bundle-DocUrl: http://www.eclipse.org
 Bundle-ContactAddress: www.eclipse.org
 Export-Package: org.osgi.service.component.annotations;version="1.3",
- org.osgi.service.log;version="1.5";uses:="org.osgi.framework";mandatory:="never.ever.use";never.ever.use="true"
+ org.osgi.service.log;never.ever.use=true;version="1.5";mandatory:="never.ever.use"
```

Comment:
Since we removed all source from this  project this is actually correct to remove the uses clause.  Consistency will still be maintained by the imported package that must resolve against the "real" export of the log package.